backends/cuda/ceed-cuda-qfunction.c

```diff
@@ -113,7 +113,7 @@ static int CeedCudaLoadQFunction(CeedQFunction qf, char *c_src_file) {
   if (!last_dot)
     return CeedError(ceed, 1, "Cannot find file's extension!");
   const size_t cuda_path_len = last_dot - cuda_file;
-  strcpy(&cuda_file[cuda_path_len], ".h");
+  strncpy(&cuda_file[cuda_path_len], ".h", 3);
```

Comment:
This technically imposes the requirement that the code is in a `.h`. Do we want to relax this and allow any extension?

This was originally added to the OCCA backend to convert `foo.c` to `foo.okl`. With the `CeedQFunction` macro we get the actual source path of the QFunction, so I think removing this would be perfectly fine.